Modules/book_module.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../classes/Book/book'
+
+module BookModule
+  def create_book(publisher, cover_state, publish_date)
+    book = Book.new(publisher, cover_state, publish_date)
+    @books << book
+    save_book_to_file(@books)
+  end
+
```

Comment:
- [ ] When creating a book there is no way to add the label "title and color" which means the label can't be saved in the `label.json` if can't create it, kindly fix that 🙏🏾